src/tests/index.test.ts

```diff
@@ -187,14 +187,14 @@ test('Child logger takes parent config', () => {
 });
 
 test('Child logger inherits parent name', () => {
-    const logger = loggerFactory('parent', { disableStackdriverFormat: true });
```

Comment:
NIT: Don't really understand this change/commit, can you explain?

Reply:
I fixed the loggerFactory type, and it does not support passing extra options to child loggers. Therefore, this argument was ignored before and now it's type-checked so I had to delete it

Reply:
Cool, thx